com.reprezen.swagedit.tests/src/com/reprezen/swagedit/assist/JsonReferenceProposalProviderTest.xtend

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ *  Copyright (c) 2016 ModelSolv, Inc. and others.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *  
+ *  Contributors:
+ *     ModelSolv, Inc. - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package com.reprezen.swagedit.assist
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.reprezen.swagedit.editor.SwaggerDocument
+import com.reprezen.swagedit.mocks.Mocks
+import org.junit.Before
+import org.junit.Test
+
+import static org.hamcrest.core.IsCollectionContaining.*
+import static org.junit.Assert.*
+
+class JsonReferenceProposalProviderTest {
+
+	val mapper = new ObjectMapper
+	var JsonReferenceProposalProvider provider
+
+	@Before
+	def void setUp() {
+		provider = new JsonReferenceProposalProvider {
+			override protected getActiveFile() {
```

Comment:
Eclipse shows a compilation error for me here:
<img width="535" alt="screen shot 2016-07-08 at 11 50 09 am" src="https://cloud.githubusercontent.com/assets/644582/16693218/3237b07a-4502-11e6-8d94-af02158b4a95.png">
I can live with it because 
1. Maven compilation succeeds
2. And it's in tests
Just  wondering if you have the same.